FlowCal/io.py

```diff
@@ -1760,6 +1760,38 @@ def __array_finalize__(self, obj):
         if hasattr(obj, '_resolution'):
             self._resolution = copy.deepcopy(obj._resolution)
 
+    # override __reduce__ and __setstate__ to fix Issue #277
+    # Adapted from https://stackoverflow.com/a/26599346
+    def __reduce__(self):
```

Comment:
We need a docstring here explaining what this function does and why it was replaced. It doesn't have to be too detailed, as this function will not be exposed to the user, but it should be enough to be useful for anyone in the future modifying this code. Please refer to https://github.com/numpy/numpy/blob/master/doc/example.py.

Reply:
[Name of return values is optional](https://numpydoc.readthedocs.io/en/latest/format.html), as they're not exposed to the method caller anyway.